lib/bolt/cli.rb

```diff
@@ -551,6 +557,114 @@ def list_groups
       outputter.print_groups(groups)
     end
 
+    def new_plan(plan_name)
+      @logger.warn("Command 'bolt plan new' is experimental and subject to breaking changes.")
```

Comment:
```suggestion
      @logger.warn("Command 'bolt plan new' is experimental and subject to change.")
```

"Breaking changes" feels like the command is dangerous, which it's not.